sys/idsueye/gstidsueyesrc.h

```diff
@@ -23,7 +23,12 @@
 #include <gst/base/gstpushsrc.h>
 
 #define _PURE_C
-#include "uEye.h"
+#ifdef _WIN32
+  #include "uEye.h"
+#endif
+#ifdef linux
+  #include "ueye.h"
+#endif
```

Comment:
Windows will ignore capitalization in filenames, so no need for the `ifdef`, just use the lower case version:
```suggestion
#include "ueye.h"
```